frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -783,7 +783,11 @@ export default ({
         if (events.length) {
           // NOTE: if 'messageHashToScroll' was not there in the messages of the contract state
           //       we need to retrieve more events, and render to scroll to that message
-          this.updateScroll(messageHashToScroll, Boolean(mhash))
+          this.updateScroll(messageHashToScroll, Boolean(mhash)).then(() => {
+            if (mhash) {
+              this.$router.replace({ query: {} })
+            }
+          })
```

Comment:
It's to remove `mhash` in queries after scroll to and highlight the message with `mhash`. Since we use only `mhash` as query in the Chat page, so just made query empty.